src/components/OrgFile/OrgFile.integration.test.js

```diff
@@ -26,15 +26,6 @@ import { STATIC_FILE_PREFIX } from '../../lib/org_utils';
 afterEach(cleanup);
 
 describe('Render all views', () => {
-  jest.mock('react-hotkeys', () => {
```

Comment:
This code was throwing errors in `yarn test`. After fixing it, I wanted to know what it really does.

Result: I think this was not needed at all. The tests are still green(;